### DISCUSSION TITLE
Cabinets won't load

### DISCUSSION THREAD

**Discussion by dschomburg** [opendcim/openDCIM#1548] [Q&A]
Setup DCIM the end of June loaded some stuff into the cabinets before I left on vacation. Everything was working fine. 

Now I come back after being gone for 10 days to a spinning green square. Reboots and rebuilding image cache don't seem to help. Cache rebuild stops and sits at 90%. Any suggestions on where to start looking?

![image](https://github.com/user-attachments/assets/ee345010-ee74-445c-9260-efbe4f4f4942)

**Comment by wilpig** [comment]
Since you didn't say what OS let's hit the usual locations and see what was installed while you were gone that could have changed something.

/var/log/dnf.rpm.log
/var/log/dpkg.log

Next i'd hit up last -n 40 and see who's been on there since you were in last and you can ask them what they did.

Failing to find a smoking gun above you will then have to do the usual things like debugging what is going on and that would involve hitting F12 on this screen and reloading the page to see some form of 500 error which will then send you to go look at 
/var/log/httpd/error_log
/var/log/httpd/ssl_error_log
/var/log/apache2/error. log
/var/log/php-fpm/error.log
/home/dev.opendcim.org/logs/opendcim_error_log

I'm going to guess you've got something borking up the API which loads the cabinet contents.  You could even have something blcoking javascript.

**Comment by dschomburg** [comment]
This is running on Ubuntu. Just real odd that it just stopped working. Below is a couple of F12 debugs. Seems that those locations are not accessible or just totally missing. Not much help in those other log files listed out either. 

![image](https://github.com/user-attachments/assets/9b17850c-e731-4986-ae9d-842cf0357acb)
![image](https://github.com/user-attachments/assets/a1df1650-24f3-48d8-a15a-3b143f09ba51)

**Comment by dschomburg** [comment]
I am thinking this has something to do with unattended automatic patching on the OS. I can see Apache updates get applied to the system.

**Comment by wilpig** [comment]
If you don't have any 500 errors in the apache log and you're just getting a 404 like this then it isn't parsing the rewrite rule correctly.

**Reply by dschomburg** [reply]
Thanks for the pointers, I am not a seasoned Linux admin/engineer. I will have to go figure out what you mean and where to start on that one. :-)

I had so little information in it, I may just reload the entire thing and start over. See if it happens again.
